numpy/core/src/_simd/_simd_inc_convert.h

```diff
@@ -0,0 +1,207 @@
+#if !NPY_SIMD
+    #error "Not a standalone header, only works through 'simd.dispatch.c.src'"
+#endif
+
+/************************************
+ ** Protected Definitions
+ ************************************/
+static simd_data
+simd_scalar_from_obj(PyObject *obj, simd_data_type dtype)
```

Comment:
For the sake of clarity:

```suggestion
simd_scalar_from_number(PyObject *obj, simd_data_type dtype)
```